frontend/src/selectors/staticContentSelectors.js

```diff
@@ -192,3 +192,43 @@ export const getVarianceStatusOptionsHash = createSelector(
   [getDropdownVarianceStatusOptions],
   createLabelHash
 );
+
+const itemInList = (child, itemToAdd) => {
+  let item = child.find((x) => x.value === itemToAdd.code);
+
+  if (item === undefined) {
+    item = { value: itemToAdd.code, label: itemToAdd.label, children: [] };
+    child.push(item);
+  }
+
+  return item;
+};
```

Comment:
Could we refactor this to be more declarative?

I'm also concerned that we are mutating `child` here. If we want to add a new item to `child`, we should almost certainly be returning a new `child` array instead.

Lastly, if `child` is an array, it should probably be `children`? Or `list`, given that the function is called `itemInList`. I'll look more at what this function is being used for, but it's raising some red flags for me

Reply:
This function ha been removed.